components/popover/tippy_utils.js

```diff
@@ -57,21 +56,17 @@ export const getPopperOptions = ({
   };
 };
 
-const createAnchor = (anchorWrapper, tabIndex) => {
+const createAnchor = (anchorWrapper) => {
   const span = document.createElement('span');
-  span.setAttribute('tabindex', tabIndex);
   span.innerText = anchorWrapper.innerText || '';
   anchorWrapper.innerText = '';
   anchorWrapper.appendChild(span);
   return span;
 };
 
-export const getAnchor = (anchorWrapper, tabIndex = '0') => {
+export const getAnchor = (anchorWrapper) => {
   const anchor = anchorWrapper?.children[0];
   if (!anchor) return createAnchor(anchorWrapper);
-  if (!findFirstFocusableNode(anchor)) {
-    anchor.setAttribute('tabindex', tabIndex);
-  }
   return anchor;
 };
```

Comment:
This was causing issues with focus in the contact row. not sure why we force tabindex=0 here, it should be focusable if the consumer makes it focusable.

Reply:
yeah seems okay so far.. haven't really tested that much though will do some more testing.

Reply:
I think this was from before when we used to have a tabIndex prop. it would set the tabIndex on the anchor to this prop. This prop was removed long ago so I think it is fine to remove.

Tested a bunch in dt-vue, doesn't seem to have broken anything.